test/test_retriever.py

```diff
@@ -113,8 +141,11 @@ def test_find_file_present():
 
     """
     test_engine.script.shortname = 'AvianBodySize'
-    os.chdir('./test/')
-    assert test_engine.find_file('avian_ssd_jan07.txt') == os.path.normpath('raw_data/AvianBodySize/avian_ssd_jan07.txt')
+    # os.chdir('test')
+    dir_path = os.path.dirname(os.path.abspath(__file__))
+    os.chdir(dir_path)
```

Comment:
Do these two lines need to be called again since they are called in `setup_module()`?